client/.env

```diff
@@ -0,0 +1 @@
+VITE_API_ENDPOINT=http://localhost:3000/v1
```

Comment:
why is this variable related to VITE? that's the api backend right? if so should be just `API_ENDPOINT` maybe

Reply:
when using vite, env vars should have VITE_API at the start. If it's not added env vars are set to undefined